resources/keyman-site.conf

```diff
@@ -0,0 +1,13 @@
+<FilesMatch .md$>
+	SetHandler text/html
+</FilesMatch>
+
+DirectoryIndex index.md index.php index.html
+
+<Directory /var/www/html>                
+	Options +Includes +FollowSymLinks -MultiViews
+	AllowOverride All
+</Directory>
+
+php_value include_path "/var/www/html/_includes:."
```

Comment:
This shouldn't be necessary as s.keyman.com is very lightweight and none of the PHP files should have includes.

We should also be removing the api/deploy.php, api/webhooks.php and App_Data/jobs/continuous/deploy/* files as they won't be needed in the docker setup. Presumably the .github/workflows will need redesign as well.

And then the only .php file of relevance is api/kmwversion.php.

Reply:
Still TODO on the .github/workflow